### ISSUE TITLE
Can I use a custom ZLib build with the curl pulled by cpr?

### ISSUE THREAD

**Issue by Iisus** [libcpr/cpr#891]
### Description

We have some non-cmake projects and I'm trying to build libcpr and publish the artifacts internally. The issue is that we already have ZLib for used by our app and we would like to use the same version for curl as well. Afaik curl supports having a custom ZLib build via `ZLIB_PATH`. 

Is it possible to build libcpr with `CPR_USE_SYSTEM_CURL=OFF`, but with our ZLib?
If not, is it possible to use our own curl build? So something like `CPR_CURL_PATH=...`?

Thank you!

### Example/How to Reproduce

This will still pull cpr's zlib:
```
cmake -DZLIB_LIBRARY=<path-to-custom-zlib>\zlib-1.2.x64.Debug.lib -DZLIB_INCLUDE_DIR=<path-to-custom-zlib>\ZLib -DCPR_USE_SYSTEM_CURL=OFF -DCMAKE_INSTALL_PREFIX=install ../cpr/
```

### Where did you get it from?

Other (specify in "Additional Context/Your Environment")

### Additional Context/Your Environment

- OS: Windows, Linux
- Version: 1.10.1
- Built from sources

**Comment by COM8** [comment]
No, this is not possible right now. But it could be implemented like it's done for `CPR_USE_SYSTEM_CURL`.
